src/ert_storage/endpoints/experiments.py

```diff
@@ -104,6 +124,35 @@ def delete_experiment(*, db: Session = Depends(get_db), experiment_id: UUID) ->
     db.commit()
 
 
+PRIOR_FUNCTION_TO_PYDANTIC: Mapping[ds.PriorFunction, Type[js.Prior]] = {
+    ds.PriorFunction.const: PriorConst,
+    ds.PriorFunction.trig: PriorTrig,
+    ds.PriorFunction.normal: PriorNormal,
+    ds.PriorFunction.lognormal: PriorLogNormal,
+    ds.PriorFunction.ert_truncnormal: PriorErtTruncNormal,
+    ds.PriorFunction.stdnormal: PriorStdNormal,
+    ds.PriorFunction.uniform: PriorUniform,
+    ds.PriorFunction.ert_duniform: PriorErtDUniform,
+    ds.PriorFunction.loguniform: PriorLogUniform,
+    ds.PriorFunction.ert_erf: PriorErtErf,
+    ds.PriorFunction.ert_derf: PriorErtDErf,
+}
+
+
+def prior_to_dict(prior: ds.Prior) -> dict:
```

Comment:
seems like this is an internal func - should it be renamed to `_prior_to_dict`?

Reply:
Sure, sounds good!